docs/src/main/asciidoc/config-yaml.adoc

```diff
@@ -55,10 +55,10 @@ quarkus:
       url: jdbc:postgresql://localhost:5432/some-database
 
 # REST Client configuration property
-org:
-  acme:
-    restclient:
-      CountriesService/mp-rest/url: https://restcountries.eu/rest
+quarkus:
+  rest-client:
+    "org.acme.restclient.CountriesService":
```

Comment:
do we need quotes here?

Reply:
No it works without quotes too.

Reply:
I removed the quotes from the example, I think it's better to keep it simpler...